core/src/test/scala/examples/package.scala

```diff
@@ -0,0 +1,20 @@
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.Sink
+
+import scala.util.Failure
+
+// Tools available for the examples
+//
+package object examples {
```

Comment:
Decided to place this in the package object (kind of trashbag for anything to be seen in a certain package - avoids needing to name an object for it).